### ISSUE TITLE
Recipe Removal for the Modpack

### ISSUE THREAD

**Issue by ghost** [Migmag789/FanaticEngMarv#6]
Different issue than suggestion for removing recipes (as opposed to altering them to make them harder).

**Comment by ghost** [comment]
Item: Bronze
Remove recipe where three copper ingots and one tin ingot is needed to make four bronze
Maybe gate bronze behind something like an alloy smelter or blast furnace

**Comment by Migmag789** [comment]
Ok, I'll do that. I'll make a recipe like 3 copper ingots and 1 tin ingot for only 2 bronze ingots so that will be better to use machines. Thank you !

**Comment by ghost** [comment]
Item: IC2 Bronze Tools & Armor
Remove, unneeded since Mekanism adds some
Item: Bronze Paxel
Remove, too OP early game

**Comment by Migmag789** [comment]
For the bronze paxel that's already done, but I think we should remove Mekanism Tools or reduce the amount of items it adds. I prefer the TE bronze tools / armor ;)

**Comment by ghost** [comment]
Sure, how do TE bronze tools/armor compare to Vanilla iron tools/armor?

**Comment by Migmag789** [comment]
Yes, I think.

**Comment by ghost** [comment]
Item: Mekanism's Bronze Tools & Armor
Remove, replace by TE bronze tools and armor

**Comment by Cosmicfox** [comment]
Yes Mekanisms tools I was a bit worried it be to OP. I love my osmium armor and tools though early game. 

I like the TE bronze idea, bronze always better then iron.. cool beans.

**Comment by ghost** [comment]
Ye Osmium needs some balancing. Perhaps we could gate osmium with something else like diamonds? For example,

 **Osmium Chestplate:**
[osmium ingot][null][osmium ingot]
[osmium ingot][diamond][osmium ingot]
[osmium ingot][osmium ingot][osmium ingot]

**Comment by Cosmicfox** [comment]
That is a marvelous idea! Thanks bro you are another great addition to the team.. Appreciate you a lot for helping out. :D
